usage/rest-api/src/main/java/brooklyn/rest/api/CatalogApi.java

```diff
@@ -202,18 +202,18 @@ public CatalogEntitySummary getEntity(
     })
     public CatalogEntitySummary getApplication(
         @ApiParam(name = "applicationId", value = "The ID of the application to retrieve", required = true)
-        @PathParam("applicationId") String entityId) throws Exception;
+        @PathParam("applicationId") String applicationId) throws Exception;
 
     @GET
-    @Path("/applications/{applicationId}/{version}")
+    @Path("/applications/{symbolicName}/{version}")
     @ApiOperation(value = "Fetch a specific version of an application's definition from the catalog", responseClass = "CatalogEntitySummary", multiValueResponse = true)
     @ApiErrors(value = {
         @ApiError(code = 404, reason = "Entity not found")
     })
     public CatalogEntitySummary getApplication(
-        @ApiParam(name = "applicationId", value = "The ID of the application to retrieve", required = true)
-        @PathParam("applicationId") String entityId,
-        
+        @ApiParam(name = "symbolicName", value = "The symbolic name of the application to retrieve", required = true)
+        @PathParam("symbolicName") String symbolicName,
+
```

Comment:
Strange, probably my Intellij not setup as it should be.

On Mon, 6 Jul 2015 at 16:37 Sam Corbett <notifications@github.com> wrote:

> In usage/rest-api/src/main/java/brooklyn/rest/api/CatalogApi.java
> <https://github.com/apache/incubator-brooklyn/pull/729#discussion_r33947593>
> :
>
> >      @ApiOperation(value = "Fetch a specific version of an application's definition from the catalog", responseClass = "CatalogEntitySummary", multiValueResponse = true)
> >      @ApiErrors(value = {
> >          @ApiError(code = 404, reason = "Entity not found")
> >      })
> >      public CatalogEntitySummary getApplication(
> > -        @ApiParam(name = "applicationId", value = "The ID of the application to retrieve", required = true)
> > -        @PathParam("applicationId") String entityId,
> > -
> > +        @ApiParam(name = "symbolicName", value = "The symbolic name of the application to retrieve", required = true)
> > +        @PathParam("symbolicName") String symbolicName,
> > +
>
> I guess you're just following the existing format? Doesn't matter, will
> merge.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-brooklyn/pull/729/files#r33947593>.
>